dev/PowerNotifications/PowerNotifications.h

```diff
@@ -7,11 +7,30 @@
 #include <powersetting.h>
 #include <Microsoft.Windows.System.Power.PowerManager.g.h>
 #include <frameworkudk\PowerNotificationsPal.h>
-#include <wil\resource.h>
-#include <wil\result_macros.h>
+#include <..\WindowsAppSDK_Insights\WindowsAppRuntimeInsights.h>
+
+class PowerNotificationsTelemetry : public wil::TraceLoggingProvider
+{
+    IMPLEMENT_TRACELOGGING_CLASS(PowerNotificationsTelemetry, "Microsoft.WindowsAppSDK.System.PowerNotifications",
+        // {a1b12e2c-12d9-564e-2ea1-2894ffcc7cc5}
+        (0xa1b12e2c, 0x12d9, 0x564e, 0x2e, 0xa1, 0x28, 0x94, 0xff, 0xcc, 0x7c, 0xc5));
+
+public:
+
+    DEFINE_COMPLIANT_TELEMETRY_EVENT_STRING(AddCallbackTrace, PDT_ProductAndServiceUsage, eventName);
```

Comment:
All of these TELEMETRY macros define a Telemetry-sampling level event. This needs to be at MEASURES sampling level so it fires. This would be the macros: DEFINE_COMPLIANT_MEASURES_EVENT_STRING (or similar).
@huzaifa-d we can work together to place the correct event sampling level for each event.

Reply:
Although, this set of events seem to complement information together, so they should all be MEASURES.

Reply:
Correct, Telemetry sampling is very seldom used. It even has to be turned on OTA, temporarily in retail, so the use case is almost extra diagnostic info after uncovering bugs. Standard use for any telemetry that we know we will consume is Measures-level. There is internal wiki information.